src/udev/udev-node.c

```diff
@@ -32,6 +32,30 @@
 #define LINK_UPDATE_MAX_RETRIES 128
 #define UDEV_NODE_HASH_KEY SD_ID128_MAKE(b9,6a,f1,ce,40,31,44,1a,9e,19,ec,8b,ae,f3,e3,2f)
 
+static int create_symlink(const char *target, const char *slink) {
+        int r;
+
+        assert(target);
+        assert(slink);
+
+        for (;;) {
```

Comment:
doesn't matter, but maybe put a limit on this? i.e. if we tried 100 times, return ENOENT after all and give up?